crypto.go

```diff
@@ -657,3 +658,37 @@ func decryptAES(password string, crypt64 string) (string, error) {
 
 	return string(decrypted[:len(decrypted)-int(decrypted[len(decrypted)-1])]), nil
 }
+
+func gzipCompress(input string) (string, error) {
+	var buffer bytes.Buffer
+	gzipWriter := gzip.NewWriter(&buffer)
+	_, err := gzipWriter.Write([]byte(input))
+	if err != nil {
+		return "", err
+	}
+
+	if err := gzipWriter.Close(); err != nil {
+		return "", err
+	}
+
+	return buffer.String(), nil
+}
+
+func gzipDecompress(input string) (string, error) {
+	gzipReader, err := gzip.NewReader(bytes.NewReader([]byte(input)))
+	if err != nil {
+		return "", err
+	}
+
+	var buffer bytes.Buffer
+	_, err = buffer.ReadFrom(gzipReader)
+	if err != nil {
+		return "", err
```

Comment:
shall we add `gzipReader.Close()` here?
Or just use `defer gzipReader.Close()` and `defer gzipWriter.Close()` in both functions without an error processing?

Reply:
I added `.Close()` using `defer` in both functions